tests/components/glances/snapshots/test_sensor.ambr

```diff
@@ -951,3 +951,50 @@
     'state': '30.7',
   })
 # ---
+# name: test_sensor_states[sensor.0_0_0_0_timestamp-entry]
```

Comment:
It appears translations was not generated given the strange name?

Reply:
You're right, and the tests weren't running on the uptime sensor.
I updated the snapshot, reference uptime and also froze time to fix the time reference used in the snapshot.